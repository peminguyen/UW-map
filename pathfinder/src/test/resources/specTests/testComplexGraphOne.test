CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D
AddNode graph E
AddNode graph F

AddEdge graph A B 7
AddEdge graph B A 7
AddEdge graph A C 9
AddEdge graph C A 9
AddEdge graph A F 14
AddEdge graph F A 14
AddEdge graph B C 10
AddEdge graph C B 10
AddEdge graph B D 15
AddEdge graph D B 15
AddEdge graph C F 2
AddEdge graph F C 2
AddEdge graph C D 11
AddEdge graph D C 11
AddEdge graph F E 9
AddEdge graph E F 9
AddEdge graph D E 6
AddEdge graph E D 6

FindPath graph A E