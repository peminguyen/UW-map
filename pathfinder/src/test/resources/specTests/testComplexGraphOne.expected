created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph
added node E to graph
added node F to graph

added edge 7.000 from A to B in graph
added edge 7.000 from B to A in graph
added edge 9.000 from A to C in graph
added edge 9.000 from C to A in graph
added edge 14.000 from A to F in graph
added edge 14.000 from F to A in graph
added edge 10.000 from B to C in graph
added edge 10.000 from C to B in graph
added edge 15.000 from B to D in graph
added edge 15.000 from D to B in graph
added edge 2.000 from C to F in graph
added edge 2.000 from F to C in graph
added edge 11.000 from C to D in graph
added edge 11.000 from D to C in graph
added edge 9.000 from F to E in graph
added edge 9.000 from E to F in graph
added edge 6.000 from D to E in graph
added edge 6.000 from E to D in graph

path from A to E:
A to C with weight 9.000
C to F with weight 2.000
F to E with weight 9.000
total cost: 20.000